lib/net.js

```diff
@@ -1130,8 +1130,12 @@ function _listen(handle, backlog) {
   return handle.listen(backlog || 511);
 }
 
-var createServerHandle = exports._createServerHandle =
-    function(address, port, addressType, fd) {
+const createServerHandle = exports._createServerHandle = function(
+  address,
+  port,
+  addressType,
+  fd
+) {
```

Comment:
No specific reason I think, except the 4-space indent is invalid there (should be 2).